docs/virtual_machines/templates.md

```diff
@@ -366,7 +412,7 @@ spec:
 ### Overview
 
 The KubeVirt projects provides a set of
-[templates](https://docs.okd.io/latest/dev_guide/templates.html) to
+[templates](https://docs.okd.io/latest/openshift_images/using-templates.html) to
```

Comment:
The OKD docs don't talk about virtual machine templates.  We should use a Red Hat OpenShift doc link right here.